api/lib/application/sessions/session-controller.js

```diff
@@ -104,9 +104,11 @@ module.exports = {
   async addCertificationCandidate(request, h) {
     const sessionId = request.params.id;
     const certificationCandidate = await certificationCandidateSerializer.deserialize(request.payload);
+    const complementaryCertifications = request.payload.data.attributes['complementary-certifications'] || [];
```

Comment:
nitpick: prendre l'habitude d'utiliser `??` plutôt que `||`
```suggestion
    const complementaryCertifications = request.payload.data.attributes['complementary-certifications'] ?? [];
```

Reply:
Bien vu